packages/smart-contracts/README.md

```diff
@@ -269,6 +269,12 @@ yarn hardhat update-contracts
 
 This command will output details about each update on each chain
 
+If the contracts are administrate by a Safe, use the flag `safe`:
+
+```bash
+yarn hardhat update-contracts --safe
```

Comment:
This should be default actually. And we should use `--eoa` to force the opposite.